generators/server/templates/src/test/java/package/web/rest/_UserResourceIntTest.java

```diff
@@ -63,10 +64,7 @@
 <%_ if (databaseType !== 'cassandra') { _%>
 import java.time.Instant;
 <%_ } _%>
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
`java.util.Arrays` and `java.util.List` is not used with OAuth; it seemed easier to import `java.util.*` than have if statements for each import.